rn/Teacher/src/modules/app/index.js

```diff
@@ -40,7 +41,7 @@ const teacher = {
 
 const student = {
   appId: 'student',
-  filterCourse: (course: Course): boolean => true,
+  filterCourse: (course: Course): boolean => !course.access_restricted_by_date,
```

Comment:
@shichongrui are you okay with this? It could possibly deviate from the dashboard cards api but I don't think we ever want to include these courses anywhere.

We talked about removing the `filterCourse` check from Dashboard (except we need it for Teacher). Turns out Dashboard uses `filterCourse` in a bunch of places so that got out of hand real quick.

Reply:
Yeah I think considering the usage of `filterCourse` it will be ok. Hopefully the dashboard api never returns courses that are access restricted cause that would seem broken.